TelenorNBIoT.cpp

```diff
@@ -369,9 +375,13 @@ bool TelenorNBIoT::receiveFrom(char *ip, uint16_t *port, char *outbuf, uint16_t
             {
                 *port = atoi(fields[2]);
             }
+            
             *length = atoi(fields[3]);
-            // TODO(stalehd): Convert data into bytes
-            strcpy(outbuf, fields[4]);
+            char *pos = fields[4];
+            pos++; // inc by 1 to skip double-quote char
+            hexToBytes(pos, *length, outbuf);
```

Comment:
This isn't a new issue, but `outbuf` needs to be big enough to hold all `*length` bytes.  Maybe `length` should also be an input parameter and should hold the size of `outbuf`, so we don't write beyond its allocation.

Reply:
Suggested fix in #14